Session/MemcachedSessionHandler.php

```diff
@@ -86,7 +86,7 @@ public function close(): bool
 	/**
 	 * {@inheritDoc}
 	 */
-	public function read($id)
+	public function read($id): string|false
```

Comment:
Just double checking this is intentional, and we're only going to publish this bundle update exactly alongside the PHP 8 upgrade? (Or should we be holding back this for a second phase to avoid the coupling on this dependency...?)

Reply:
It's a good question, I figure we'd do it all together.